src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -261,13 +261,14 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
             ref CustomAttributesBag<CSharpAttributeData> lazyCustomAttributesBag,
             AttributeLocation symbolPart = AttributeLocation.None,
             bool earlyDecodingOnly = false,
-            Binder binderOpt = null)
+            Binder binderOpt = null,
+            Func<AttributeSyntax, bool> attributeMatchesOpt = null)
```

Comment:
>Func<AttributeSyntax, bool> attributeMatchesOpt = null [](start = 12, length = 54)

Is the lack of ```<param ...>...``` tag is causing a warning during compilation?

Reply:
It didn't, but I'll add doc to match other parameters.

---
In reply to: [155960781](https://github.com/dotnet/roslyn/pull/23647#discussion_r155960781) [](ancestors = 155960781)